airflow/models/dag.py

```diff
@@ -296,8 +298,16 @@ def __init__(
         self.max_active_runs = max_active_runs
         self.dagrun_timeout = dagrun_timeout
         self.sla_miss_callback = sla_miss_callback
-        self._default_view = default_view
```

Comment:
I'm not really sure why we use `self._default_view` as property name before? Prevent cover the Flask `default_view` variable ?

Reply:
I have no idea tbh, @vardancse @milton0825 - Do you guys have any idea about it?

Reply:
no idea either...

Reply:
So I think is ok to change to `default_view `